packages/rs-drive/src/drive/shared_estimation_costs/add_estimation_costs_for_contested_document_tree_levels_up_to_contract/v0/mod.rs

```diff
@@ -130,7 +131,25 @@ impl Drive {
                 ),
                 EstimatedLayerInformation {
                     is_sum_tree: false,
-                    estimated_layer_count: ApproximateElements(document_type_count),
+                    estimated_layer_count: ApproximateElements(2),
+                    estimated_layer_sizes: AllSubtrees(
+                        ESTIMATED_AVERAGE_INDEX_NAME_SIZE,
+                        NoSumTrees,
+                        None,
+                    ),
+                },
+            );
+
+            estimated_costs_only_with_layer_info.insert(
```

Comment:
do you insert `estimated_costs_only_with_layer_info` twice for purpose?

Reply:
yes, they are not the same